cmd/kubeapps-apis/proto/kubeappsapis/plugins/fluxv2/packages/v1alpha1/fluxv2.proto

```diff
@@ -127,3 +127,19 @@ message SetUserManagedSecretsRequest {
 message SetUserManagedSecretsResponse {
   bool value = 1;
 }
+
+// Flux PackageRepositoryCustomDetail
+//
+// Custom details for a Flux Package repository
+message FluxPackageRepositoryCustomDetail {
+  // optional field that allows specifying an OIDC provider used for authentication purposes
+  // Supported options are:
+  //  - generic
+  //  - aws
+  //  - azure
+  //  - gcp
+  // The provider field is supported only for Helm OCI repositories. The repository type must 
+  // be set to "oci"
+  // ref https://fluxcd.io/flux/components/source/helmrepositories/#provider
+  string provider = 1;
```

Comment:
Given that it is a fixed set of options, why not using [a protobuf `enum`](https://developers.google.com/protocol-buffers/docs/proto3#enum)?

Reply:
these come from flux. I don't want to repeat their definitions and I want to let them validate the string values